js/tests/unit/tab.spec.js

```diff
@@ -630,6 +630,108 @@ describe('Tab', () => {
       expect(spyPrevent).toHaveBeenCalledTimes(2)
     })
 
+    it('if keydown event is Home, handle it', () => {
```

Comment:
Wouldn't be a bit simpler to 'show' programmatically the second link, and after this , trigger this HOME btn, checking a spy or the 'show' event ?

Reply:
I modeled the test to be similar to the existing ones.  I can change this so it's a bit simpler

Reply:
Done @GeoSot